src/plugins/data/common/field_formats/static.ts

```diff
@@ -17,19 +17,15 @@
  * under the License.
  */
 
-/**
- * Everything the file exports is public
- */
```

Comment:
Removing this implies that this file now exports some things that are internal - is that the case?

Reply:
Well, it exports just one thing, and we don't import * from it anymore.
So I found the comment irrelevant.

Reply:
My overall vote would be to consider the direction I mentioned about composing these objects directly in the `public/index` and `server/index` files, in which case it doesn't matter.

That said, if we were to keep this implementation with `static.ts` I would think it would be important to identify somewhere in the file that this entire namespace is exported publicly, so you don't run the risk of someone adding something internal here accidentally.

Reply:
@lukeelmers I update the PR to constructing the namespace in public\server index.ts.